lib/kaguya/core.ex

```diff
@@ -68,6 +68,17 @@ defmodule Kaguya.Core do
     {:noreply, %{socket: socket}}
   end
 
+  ## Active mode errors
+  def handle_info({:tcp_error, _socket, reason}, state) do
+    Logger.error fn -> "TCP Socket Error happend. Reason: #{reason}" end
```

Comment:
I understand your reasoning, but for the sake of keeping consistency with prior logging code I'd prefer it remain a formatted string